### ISSUE TITLE
Web client doesn't display any datetime field when language is set to "zh-TW - 中文（台灣）"

### ISSUE THREAD

**Issue by andypengtw** [bitwarden/clients#5220]
### Steps To Reproduce

1. Go to "Account settings"->"Preferences".
2. Set "Language" to "zh-TW - 中文（台灣）" and save.
3. Re-login to web vault.
4. Check any entry, all the datetime fields are empty now.

### Expected Result

Datetime fields should display value when language is set to "zh-TW - 中文（台灣）".

### Actual Result

Datetime fields are missing values when language is set to "zh-TW - 中文（台灣）".

### Screenshots or Videos

"zh-TW - 中文（台灣）"
![cht](https://user-images.githubusercontent.com/121341724/232176698-6038e00b-3cdb-4b16-8552-0e1887592fef.PNG)

"en - English"
![eng](https://user-images.githubusercontent.com/121341724/232176710-7b1de865-ab56-48f6-80a2-4e3b2fe02127.PNG)

### Operating System

Windows

### Operating System Version

Windows 10 22H2

### Web Browser

Microsoft Edge, Firefox

### Browser Version

Edge 112.0.1722.39, Firefox 112.0

### Build Version

2023.3.0

### Issue Tracking Info

- [ ] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SaabirahMohamed** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

Bitwarden Customer Success team

**Comment by sunsky131221** [comment]
Hi All,
I found a function called fido2CredentialCreationDateValue may cause this issue.
But I'm not good in angular, somebody can help?

**Comment by sunsky131221** [comment]
研究出問題所在，以下是暫時解方瑀問題聚焦：
修改擴充套件中./popup/main.js
將
```typescript
{get fido2CredentialCreationDateValue(){var e,t,i,n;return`${this.i18nService.t("dateCreated")} ${this.datePipe.transform(null===(n=null===(i=null===(t=null===(e=this.cipher)||void 0===e?void 0:e.login)||void 0===t?void 0:t.fido2Credentials)||void 0===i?void 0:i[0])||void 0===n?void 0:n.creationDate,"short")}`}
``` 
取代為
```typescript
{get fido2CredentialCreationDateValue(){var e,t,i,n;return`${this.i18nService.t("dateCreated")}`}
``` 


以及
修改擴充套件中./popup/main.js.map
將
```typescript
get fido2CredentialCreationDateValue(): string {\n    const dateCreated = this.i18nService.t(\"dateCreated\");\n    const creationDate = this.datePipe.transform(\n      this.cipher?.login?.fido2Credentials?.[0]?.creationDate,\n      \"short\",\n    );\n    return `${dateCreated} ${creationDate}`;\n  }
``` 
取代為
```typescript
get fido2CredentialCreationDateValue(): string {\n    const dateCreated = this.i18nService.t(\"dateCreated\");\n    return `${dateCreated}`;\n  }
``` 

透過註銷view.component.ts與add-edit.component.ts中
在
```typescript
  get fido2CredentialCreationDateValue(): string {
    const dateCreated = this.i18nService.t("dateCreated");
    const creationDate = this.datePipe.transform(
      this.cipher?.login?.fido2Credentials?.[0]?.creationDate,
      "short",
    );
    return `${dateCreated} ${creationDate}`;
  }
``` 
裡面包著的
```typescript
    const creationDate = this.datePipe.transform(
      this.cipher?.login?.fido2Credentials?.[0]?.creationDate,
      "short",
    );
``` 
與return的
```typescript
${creationDate}`;
  }
``` 
就可以達到規避DatePipe錯誤。

所以可以得知在DateFormat經過 angular 的 locale 轉換應是出了問題，這就需要更專業的人協助了。
目前只能停用CreationDate資料的傳入以使擴充套件避免DatePipe錯誤導致無法render Passkey以下的欄位。

**Comment by sunsky131221** [comment]
Edit view.component.ts and add-edit.component.ts 

change these
```typescript
  get fido2CredentialCreationDateValue(): string {
    const dateCreated = this.i18nService.t("dateCreated");
    const creationDate = this.datePipe.transform(
      this.cipher?.login?.fido2Credentials?.[0]?.creationDate,
      "short",
    );
    return `${dateCreated} ${creationDate}`;
  }
``` 

to

```typescript
  get fido2CredentialCreationDateValue(): string {
    const dateCreated = this.i18nService.t("dateCreated");
    const creationDate = this.datePipe.transform(
      this.cipher?.login?.fido2Credentials?.[0]?.creationDate,
      "shortDate",
    );
    return `${dateCreated} ${creationDate}`;
  }
``` 

Will fix this issue.

**Comment by rick890101** [comment]
Hello everyone,

Earlier, I reported a similar issue through a Bitwarden Support email (Case #543010).

Up to this point, the same problem persists without resolution. Attached is the current situation along with my suggestion.

![QMjD8Us](https://github.com/user-attachments/assets/420e8f6f-d4c6-4d1e-bdb1-f1ba4de4a23e)
![aTkL7Tv](https://github.com/user-attachments/assets/52ec57bf-5fe5-40c4-badf-0b7eae0df903)
![UeWAb44](https://github.com/user-attachments/assets/e9790a36-75bb-4b96-b5ab-9e40a739270f)

### Operating System

Windows 11, Debian, more...?

### Operating System Version

Windows 11 24H2, Debian 12.7, more...?

### Web Browser

Google Chrome

### Browser version

Chrome 130.0.6723.59

### Build Version

Chrome extension 2024.10.0

The issue with the date not displaying occurs in the zh_Hant/Traditional Chinese interface. However, upon testing, the date displays correctly in the zh_Hans/Simplified Chinese interface.

I suggest copying the date display functionality from Simplified Chinese to Traditional Chinese for proper rendering.

**Comment by rick890101** [comment]
![image](https://github.com/user-attachments/assets/8ee0c969-0541-40e3-b854-ad1ef65148af)
![image](https://github.com/user-attachments/assets/a62dac5f-18e0-406d-a847-60c81ebd1d32)

It is still related to the Traditional Chinese interface. In the current version, login information with Passkey is unable to generate new passwords, resulting in an endless loading circle.

This issue has persisted for a long time and seems to be affecting normal functionality. I'm already considering switching to a competitor.

### Build Version

Chrome extension 2024.10.1

**Comment by andypengtw** [comment]
For a long time, the browser client has had fundamental issues in the Traditional Chinese environment. Honestly, I would rather the official remove the broken Traditional Chinese support for the browser client. Using the English version client by default in the Traditional Chinese environment would probably provide a better user experience, since there is currently no way to change the browser client language other than changing the browser language settings.

To put it simply, if you are a Traditional Chinese user, it is more practical to consider other password management solutions first.

**Comment by Ayx03** [comment]
I posted a feature request `Allow manual language selection in the Bitwarden extension` in the Bitwarden Community Forum: <https://community.bitwarden.com/t/allow-manual-language-selection-in-the-bitwarden-extension/76311>
